### DISCUSSION TITLE
CDATA?

### DISCUSSION THREAD

**Discussion by tolgaulas** [downforcetech/fluidxml.php#4] [Questions]
Great work. When do you plan for the CDATA nodes? (?Perhaps a workaround with domdocument?)

**Comment by daniele-orlando** [comment]
Thank you!
Cdata support is already there. 
Check the `->appendCdata()` method and its concise alias `->cdata()`.

Example:
`fluidxml()->cdata("Cdata can contain < && > special chars");`

**Comment by tolgaulas** [comment]
Yey! Maybe documentation is missing. Thank you.

Best regards.

## 

Tolga Ulas
+905337464908, skype:tolgaulas

**Comment by daniele-orlando** [comment]
All available methods are listed inside `documents/APIs.md`.

I missed to include `->cdata()`, but `->appendCdata()` together with `->appendSibling()`, `->prependSibling()` etc. are listed there.
